src/MarketingPageAcceptanceTestsSpecflow/Steps/AboutSolution/LastUpdatedDate.cs

```diff
@@ -39,8 +39,14 @@ public void WhenTheContentHasBeenUpdated()
             _context.Pending();
         }
         
-        [Then(@"the Last Changed Date is updated")]
-        public void ThenTheLastChangedDateIsUpdated()
+        [Then(@"the Last Changed Date is updated in the SolutionDetail table")]
+        public void ThenTheLastChangedDateIsUpdatedInSolutionDetail()
+        {
+            _context.Pending();
+        }
+
+        [Then(@"the Last Changed Date is updated in the MarketingContact table")]
+        public void ThenTheLastChangedDateIsUpdatedInMarketingContact()
```

Comment:
Can these 2 methods be parameterised?

Reply:
```
[Then(@"the Last Changed Date is updated in the (.*) table")]
public void ThenTheLastChangedDateIsUpdated(string tableName)
```

Reply:
totally, would you prefer i did this now or during implementation?

Reply:
Prefer now, then we can estimate better how long it's going to take and write the SQL correctly from the off